src/app/features/request-info/request-info.component.ts

```diff
@@ -27,4 +28,21 @@ export class RequestInfoComponent {
   processOnClick() {
     console.log('processOnClick');
   }
+  filterCountry(event: any){
+    console.log("filter country",event);
+    this.countrySuggestions=[];
+    // this.countrySuggestions= this.countries;
```

Comment:
remove this commented code

Reply:
I kept this comment wantedly as I am trying to compare regular for loop with advanced filters in arrays. I will remove in the future commits.